translations/stripes-erm-components/en.json

```diff
@@ -102,49 +102,69 @@
   "errors.uploadError.1": "<strong>Error:</strong> The file was not uploaded because the connection was dropped.",
   "errors.uploadError.2": "Please check the filesize is <strong>{number}MB or less</strong> and try again.",
   "errors.uploadError.3": "If the problem persists contact your system administrator.",
-  "errors.termNameHasNonAlpha": "The term name can only contain Latin alphabetical characters (a-z) and non-leading numeric characters.",
+  "errors.hasNonAlphaTermName": "The term name can only contain Latin alphabetical characters (a-z) and non-leading numeric characters.",
```

Comment:
The following isn't a big deal, I mention it because of the previous comments about terms/Terms:

Having separate translations for a message like this isn't really important. We could remove the "term" from this message and it's still completely obvious what we're talking about because it's next to the `Name` field. So if we only have an `errors.hasNonAlphaName` translation, we don't need to check which translation to render based on whether we're rendering terms/custom properties. So it may actually let you avoid having to pass props or do the `terms -> Terms` conversion, etc, basically cleaning things up.

Reply:
Will cleanup as a part of the final skim through the PR. Thanks